metagpt/tools/swe_agent_commands/_setup_default_env.sh

```diff
@@ -0,0 +1,20 @@
+# _setup_default_env.sh
+# Default Mode from SWE-Bench
+# https://github.com/princeton-nlp/SWE-agent/blob/ca54d5556b9db4f4f2be21f09530ce69a72c0305/config/configs/default_sys-env_window100-detailed_cmd_format-last_5_history-1_demos.yaml
```

Comment:
Please check the corresponding license and indicate whether it involves license declaration issues. If so, please declare it in the appropriate place.